t/mime-header.t

```diff
@@ -136,6 +136,7 @@ while (my ($e, $d) = splice @splice, 0, 2) {
     is Encode::decode('MIME-Header', $e) => $d, info("decode default: $e => $d");
 }
 
+no warnings 'once';
```

Comment:
Warning is thrown there because there is bug in test :-) And muting that warning is not a solution.